### ISSUE TITLE
how to get distilbert-base-uncased-distilled-squad?

### ISSUE THREAD

**Issue by RyanHuangNLP** [huggingface/transformers#1193]
## ❓ Questions & Help

I think it is initialize a six layer bert and distill with a 12 layer bert, then save the checkpoint file, is it right?

**Comment by VictorSanh** [comment]
Hello @RyanHuangNLP,
I am not sure to get your question.
Do you mean that you want to play with the model (do inferences)? If so, did you try `qa_model = DistilBertForQuestionAnswering.from_pretrained('distilbert-base-uncased-distilled-squad')`?

**Comment by RyanHuangNLP** [comment]
@VictorSanh I wonder how to get the 'distilbert-base-uncased-distilled-squad' pretrain model, just use the first six layer of the base one or initialize a six layer bert?

**Comment by thomwolf** [comment]
If you do:
```
qa_model = DistilBertForQuestionAnswering.from_pretrained('distilbert-base-uncased-distilled-squad')
```
You will get the 'distilbert-base-uncased-distilled-squad' pretrain model.
Nothing more to do.

**Comment by RyanHuangNLP** [comment]
I am sorry not clearly expressed, my question is how to get the 'distilbert-base-uncased-distilled-squad' pretrain model, there is no six layer bert release

**Comment by VictorSanh** [comment]
Ok, I understand your question now @RyanHuangNLP.
It is finetuned from `distilbert-base-uncased`. More precisely, the model we release is finetuned AND distilled at the same time: the loss is computed from the classic qa loss (see `run_squad.py` and `DistilBertForQuestionAnswering`) plus the distillation supervision from a BERT SQuAD model (second loss).
We haven't released the script for doing that (and we plan to do it in the near future) but it is a simple adaptation of `run_squad.py` (mostly adding a second loss i.e. distillation).

**Comment by marcoaleixo** [comment]
do we have any release date of that run_squad_adapted.py @VictorSanh ? :D
